### PR TITLE
Add back LightInject section

### PR DESCRIPTION
**Pull request by sofietoft**

### CODE REVIEW

File: Reference/Using-Ioc/index.md [hunk 1]
```diff
@@ -189,6 +189,21 @@ namespace Example.Core.Components
...
 
+### Accessing LightInject container
+
+Should you need to carry out more complicated registrations beyond the minimalist Umbraco DI implementation, you can access the underlying DI container via the `Concrete` property of the `composition`.
+
+```csharp
+var container = composition.Concrete as LightInject.ServiceContainer;
+container.Register<IFoo, Foo>();
+
+// It's not currently possible to do the following without some workarounds
```
Comment (rustybox) [inline Reference/Using-Ioc/index.md]: I think maybe this

```csharp
// It's not currently possible to assembly scan without some workarounds
// see https://github.com/umbraco/Umbraco-CMS/issues/7502 for details
// container.RegisterAssembly(/* Any method signature */);
```

**Comment by sofietoft** [comment]
Thanks for the suggestion @rustybox ! 💪 

I've applied. Let me know what you think, and we can get this merged today!

**Comment by rustybox** [comment]
Looks good to me, much less likely to cause confusion!

**Comment by sofietoft** [comment]
Brilliant. Thanks for the report and for helping out @rustybox ! 😄 

I'll get this merged right away.

**Pull request merged**